lib/xapi/implementation.rb

```diff
@@ -102,6 +104,13 @@ def readme_body
       ].reject(&:empty?).join("\n").strip
     end
 
+    def incomplete_solutions_body
+      <<-README
+## Submitting Incomplete Problems
+It's possible to submit an incomplete solution so you can see how others have completed the exercise.
+      README
+    end
```

Comment:
@gusseting suggested putting this in a text file, but I don't think that's necessary. All the other things come from text files because they live in other repositories. Putting this in a text file somewhere introduces unnecessary indirection for the sole purpose of making it look like the other things. I'd rather go with the simple solution and complicate it if necessary.